CHANGELOG.asciidoc

```diff
@@ -3,6 +3,442 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.7.0]]
+=== Beats version 7.7.0
+https://github.com/elastic/beats/compare/v7.6.2...v7.7.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Variable substitution from environment variables is not longer supported. {pull}15937{15937}
+- Change aws_elb autodiscover provider field name from elb_listener.* to aws.elb.*. {issue}16219[16219] {pull}16402{16402}
+- Remove `AddDockerMetadata` and `AddKubernetesMetadata` processors from the `script` processor. They can still be used as normal processors in the configuration. {issue}16349[16349] {pull}16514[16514]
+
+*Journalbeat*
+
+- Improve parsing of syslog.pid in journalbeat to strip the username when present {pull}16116[16116]
+
+*Metricbeat*
+
+- Add new dashboard for VSphere host cluster and virtual machine {pull}14135[14135]
+
+*Packetbeat*
+
+- Added redact_headers configuration option, to allow HTTP request headers to be redacted whilst keeping the header field included in the beat. {pull}15353[15353]
```

Comment:
Does not sound like a breaking change.

Reply:
I agree.  I moved it to the added section.